packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -104,27 +119,29 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
                 minIncludeTextWidth: 800,
               ),
             ),
+            summaryTreeController?.isExperimentalStoryOfLayoutEnabled == true
+              ? OutlineButton(
+              onPressed: summaryTreeController?.toggleDebugLayoutSummaryEnabled,
+              child: Label(
+                FlutterIcons.lightbulb,
+                'Layout',
+                minIncludeTextWidth: 900,
+              ))
+              : const SizedBox(),
             const Spacer(),
             Row(children: getServiceExtensionWidgets()),
           ],
         ),
         Expanded(
           child: Row(
             children: [
+              summaryTree,
               Expanded(
-                  child: InspectorTree(
-                controller: summaryTreeController,
-              )),
-              Expanded(
-                child: Stack(
-                  children: [
-                    InspectorTree(
-                      controller: detailsTreeController,
-                    ),
-                    if (_expandCollapseSupported) _expandCollapseButtons()
-                  ],
-                ),
-              )
+                child: detailsTreeController?.isExperimentalStoryOfLayoutEnabled == true ?
+                  InspectorDetailsTabController(
```

Comment:
+1 on this. we also need to figure out how to keep the tab buttons from being too close to the expand-collapse buttons. 
Any ideas @kenzieschmoll? For example, we could move those buttons to be aligned with the tab bar and/or switch them to icons. There should be plenty of existing expand-collapse icons we can steal.

Reply:
moving them to the tab bar sgtm. I don't know what the "Layout Details" view is going to look like, but if the expand/collapse buttons do not apply to that view, we'd need to hide them for that case.

Reply:
Adding bottom padding to the Controller will also impact on the Tree itself. Did you mean the Tab Bar before the Tab View itself?
I will try to align the tab bar with the tab bar later

Reply:
From your updated screenshots, it doesn't seem like the padding is quite like I meant. There still isn't space between the details tree and the green line that indicates a tab is selected. I meant this:
```
Details Tree
____________    <-- green selection line
[Padding here]
[tree view]
```
What it looks like right now is:
```
Details Tree
[Padding here]
____________    <-- green selection line
[tree view]
```

Reply:
I have tried aligning the expand/collapse buttons to the TabBar and adding padding to this row, what do you think? @kenzieschmoll  @jacob314 

![image](https://user-images.githubusercontent.com/25892469/67712428-2fd38700-f981-11e9-9008-16eb05e89fe1.png)

Reply:
The padding before the green selection line is actually Flutter's default, I'm currently looking into that since I'm still unsure on how to set that padding properly, I tried setting `indicatorPadding`/`labelPadding` in the `TabBarView` but both of them don't seem to work :'(

Reply:
All I was concerned with was having padding between the green selection line and the first node in the tree, so I'm happy with this! 👍  looks good!

Reply:
Thanks Kenzie!
There's once concern with the current implementation when scrolling the details tree though, since I'm giving the padding from the TabBar instead of the ScrollView, when scrolling the details tree, the animation physics shows something like this:

![Peek 2019-10-28 19-56](https://user-images.githubusercontent.com/25892469/67734034-3a137680-f9bd-11e9-821b-37235eb07aef.gif)

Should I add the padding/margin from inside the ScrollView instead? (The first element in the Inspector tree)

What do you think? @kenzieschmoll  @jacob314

Reply:
I agree that isn't good. I would add the padding to the ScrollView instead. You will likely need to tweak the logic to compute the location of a specific node in the FlutterTreeView to reflect the padding but it should be a small change.